mlflow/tracking/Dockerfile

```diff
@@ -0,0 +1,7 @@
+# example build command docker build . --build-arg TAG=v1.28.0
```

Comment:
Can we create a new directory in the repository root and move this file there?

Reply:
Is there an organization benefit to this as opposed to leaving the Dockerfiles under the related components? I am not opposed to making this change I am just curious what the expected benefit is.

Reply:
Does this Dockerfile interact with files in `mlflow/tracking`?

Reply:
I'd put `Dockerfile` in `mlflow/tracking` if it was used in `mlflow/tracking`.

Reply:
I think it's cleaner if we organize files like this:

```
- docker
  - Dockerfile
  - README.md
  - compose.yml
```

Reply:
we actually don't use any of the files in mlflow tracking. I will make this change.